dev/com.ibm.websphere.org.eclipse.microprofile/config.1.4.bnd

```diff
@@ -14,8 +14,10 @@ bVersion=1.0
 Bundle-SymbolicName: com.ibm.websphere.org.eclipse.microprofile.config.1.4; singleton:=true
 
 Import-Package: \
-  javax.enterprise.util; version="[1.1,3)",\
+  javax.enterprise.util; version="[1.1,3)"; resolution:=optional,\
+  javax.inject; version="[1.0,2)"; resolution:=optional,\
```

Comment:
Why do we add this package? MP Config does not need this package.

Reply:
@tevans78 explained that it was in bnd import-package. I understood adding here is to add optional resolution.